### PR TITLE
Add option to unselect garp in resource_aci_fvbd

### PR DESCRIPTION
**Pull request by shrsr**
1. Add option "disable" which unselects "garp". 
2. Though "disable" is used as the attribute, "" is being used in the payload. So "disable" is like an alias.
Example:
  ep_move_detect_mode            = ["garp"]
or
  ep_move_detect_mode            = ["disable"]

Includes updated doc and example.
Tried and tested.

### CODE REVIEW

File: aci/resource_aci_fvbd.go [hunk 1]
```diff
@@ -69,15 +69,18 @@ func resourceAciBridgeDomain() *schema.Resource {
 				}, false),
 			},
 
-			"ep_move_detect_mode": &schema.Schema{
```
[the remark this answers is not in the document]
Comment (shrsr) [reply aci/resource_aci_fvbd.go]: Agreed. Made a simple fix.

File: examples/bridge_domain/bd.tf [hunk 2]
```diff
@@ -11,7 +11,7 @@ resource "aci_bridge_domain" "demobd" {
   optimize_wan_bandwidth         = "no"
   arp_flood                      = "no"
   ep_clear                       = "no"
-  ep_move_detect_mode            = "garp"
+  ep_move_detect_mode            = ["garp"]
```
Comment (lhercot) [inline examples/bridge_domain/bd.tf:14]: This is not necessary anymore right?

File: website/docs/r/bridge_domain.html.markdown [hunk 3]
```diff
@@ -24,13 +24,13 @@ Manages ACI Bridge Domain
...
 		tenant_dn                   = aci_tenant.tenant_for_bd.id
-		description                 = "from terraform"
+		description                 = "created from terraform"
 		name                        = "demo_bd"
 		optimize_wan_bandwidth      = "no"
 		annotation                  = "tag_bd"
 		arp_flood                   = "no"
 		ep_clear                    = "no"
-		ep_move_detect_mode         = "garp"
+		ep_move_detect_mode         = ["garp"]
```
Comment (lhercot) [inline website/docs/r/bridge_domain.html.markdown:33]: Shouldn't it be
```
 ep_move_detect_mode = "garp"

File: website/docs/r/bridge_domain.html.markdown [hunk 4]
```diff
@@ -50,34 +50,33 @@ Manages ACI Bridge Domain
...
-- `name_alias` - (Optional) Name alias for object bridge domain.
-- `bridge_domain_type` - (Optional) The specific type of the object or component. Allowed values are "regular" and "fc". Default is "regular".
-- `unicast_route` - (Optional) The forwarding method based on predefined forwarding criteria (IP or MAC address). Allowed values are "yes" and "no". Default is "yes".
-- `unk_mac_ucast_act` - (Optional) The forwarding method for unknown layer 2 destinations. Allowed values are "flood" and "proxy". Default is "proxy".
-- `unk_mcast_act` - (Optional) The parameter used by the node (i.e. a leaf) for forwarding data for an unknown multicast destination. Allowed values are "flood" and "opt-flood". Default is "flood".
-- `v6unk_mcast_act` - (Optional) M-cast action for object bridge_domain. Allowed values are "flood" and "opt-flood". Default is "flood".
-- `vmac` - (Optional) Virtual MAC address of the BD/SVI. This is used when the BD is extended to multiple sites using l2 Outside. Only allowed values is "not-applicable". Default value is "not-applicable".
+## Argument Reference ##
```
Comment (lhercot) [inline website/docs/r/bridge_domain.html.markdown:53]: Why all those changes?

File: website/docs/r/bridge_domain.html.markdown [hunk 5]
```diff
@@ -50,34 +50,33 @@ Manages ACI Bridge Domain
...
+* `tenant_dn` - (Required) Distinguished name of parent Tenant object.
+* `name` - (Required) name of Object bridge_domain.
+* `optimize_wan_bandwidth` - (Optional) Flag to enable OptimizeWanBandwidth between sites. Allowed values are "yes" and "no". Default is "no".
+* `annotation` - (Optional) annotation for object bridge_domain.
+* `arp_flood` - (Optional) A property to specify whether ARP flooding is enabled. If flooding is disabled, unicast routing will be performed on the target IP address. Allowed values are "yes" and "no". Default is "no".
+* `ep_clear` - (Optional) Represents the parameter used by the node (i.e. Leaf) to clear all EPs in all leaves for this BD. Allowed values are "yes" and "no". Default is "no".
+* `ep_move_detect_mode` - (Optional) The End Point move detection option uses the Gratuitous Address Resolution Protocol (GARP). A gratuitous ARP is an ARP broadcast-type of packet that is used to verify that no other device on the network has the same IP address as the sending device.
+Allowed values: "garp" and "disable". Default is "disable". Type - List.
```
Comment (lhercot) [inline website/docs/r/bridge_domain.html.markdown:61]: Not type list anymore

**Review by lhercot — CHANGES_REQUESTED** [review]

File: examples/bridge_domain/bd.tf (hunk 2 above)
Comment (shrsr) [reply examples/bridge_domain/bd.tf]: Yes

File: website/docs/r/bridge_domain.html.markdown (hunk 3 above)
Comment (shrsr) [reply website/docs/r/bridge_domain.html.markdown]: Yes

File: website/docs/r/bridge_domain.html.markdown (hunk 4 above)
Comment (shrsr) [reply website/docs/r/bridge_domain.html.markdown]: Reverted

File: website/docs/r/bridge_domain.html.markdown (hunk 5 above)
Comment (shrsr) [reply website/docs/r/bridge_domain.html.markdown]: Yes

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**